Trailheads/src/classes/AnimalLocatorMock.cls

```diff
@@ -0,0 +1,15 @@
+@isTest
+global class AnimalLocatorMock implements HttpCalloutMock {
+    public static final String animalName = 'chicken';
+    global HttpResponse respond(HTTPRequest request) {
+        String animalId = request.getEndpoint().substringAfterLast('/');
+        AnimalLocator.Animal animal = new AnimalLocator.Animal(animalId, animalName, 'chicken food', 'cluck cluck');
+        System.debug('jkub // ' + request);
```

Comment:
staramy się nie wrzucać do repo takich rzeczy - to jest widziane ako śmiecenie raczej